spec/mongo/bulk_write_spec.rb

```diff
@@ -1914,6 +1914,8 @@
 
             let(:subscriber) { EventSubscriber.new }
 
+            let(:subscriber) { EventSubscriber.new }
```

Comment:
I assume this being duplicated is a mistake

Reply:
Yes indeed.